File: Java/Medium/Accepted/minimum-operations-to-make-array-equal_6.java
class Solution {
    public int minOperations(int n) {
        
        int[] arr = new int[n];
        for(int i = 0; i < n; i++) {
            arr[i] = (2 * i) + 1;
        }
        int target = 0;
        for(int i = 0; i < n; i++) {
            target += arr[i];
        }
        target /= n;
        int count = 0;
        for(int i = 0; i < n / 2; i++) {
            count += target - arr[i];
        }
        return count;
    }
}

File: Rust/Medium/CompileError/minimum-operations-to-make-array-equal_6.rs
impl Solution {
    pub fn min_operations(n: i32) -> i32 {
        
        let n = n as i64;
        let mut sum = 0;
        for i in 0..n/2 {
            sum += n - 2*i - 1;
        }
        sum
    }
}